lib/utils/ensureOption.js

```diff
@@ -8,8 +8,7 @@
 function defaults(defaultOptions) {
   return function ensureOption(options, name, defaultValue) {
     let value;
-
-    if (typeof options[name] !== 'undefined') {
+    if (typeof options[name] !== 'undefined' || name === 'oauth_token') {
```

Comment:
What does this do specifically?

Reply:
this sets 'oauth_token' if it not undefined